update_item.go

```
@@ -0,0 +1,84 @@
+		Key:              t.NewKeys(pk, sk),
+		UpdateExpression: updateExpression,
+	}
+
+	if len(expressionAttributeValues) > 0 {
+		input.ExpressionAttributeValues = expressionAttributeValues
+	}
+
+	// Apply option functions
+	if len(opts) > 0 {
+		for _, opt := range opts {
+			opt(input)
```

golangci-lint (2.12.2)

[error] Error return value is not checked

(errcheck)